src/components/UserProfile/AddNewUserProfile/UserProfileAdd.jsx

```diff
@@ -488,7 +508,21 @@ class AddUserProfile extends Component {
     }
   };
 
-  createUserProfile = () => {
+  checkIfDuplicate = (firstName, lastName) => {
+    let { userProfiles } = this.state.userProfiles;
+
+    const duplicates = userProfiles.filter(user => {
+      return (
+        user.firstName.toLowerCase() == firstName.toLowerCase() &&
```

Comment:
=== should be used for comparison instead of ==

Reply:
Can't we use find instead of filter ? Filter will run for entire length of userProfiles where as find will stop at first instance adding to performance.